include/fmt/format.h

```diff
@@ -1871,22 +1879,75 @@ auto write_float(OutputIt out, const DecimalFP& fp,
   });
 }
 
+#ifdef __cpp_lib_bit_cast
+template <typename T, FMT_ENABLE_IF(is_fast_float<T>::value)>
+constexpr bool is_infinite_fast_float(T value) {
+  using floaty =
+      std::conditional_t<std::is_same<T, long double>::value, double, T>;
+  using uint = typename dragonbox::float_info<floaty>::carrier_uint;
+  constexpr auto significand_bits =
+      dragonbox::float_info<floaty>::significand_bits;
+  auto bits = std::bit_cast<uint>(value);
+  return (bits & exponent_mask<floaty>()) &&
+         !(bits & ((uint(1) << significand_bits) - 1));
+}
+
+template <typename T, FMT_ENABLE_IF(is_fast_float<T>::value)>
+constexpr bool is_finite_fast_float(T value) {
+  using floaty =
+      std::conditional_t<std::is_same<T, long double>::value, double, T>;
+  using uint = typename dragonbox::float_info<floaty>::carrier_uint;
+  auto bits = std::bit_cast<uint>(value);
+  return (bits & exponent_mask<floaty>()) != exponent_mask<floaty>();
+}
+#endif
+
+template <typename T, FMT_ENABLE_IF(std::is_floating_point<T>::value)>
+FMT_INLINE FMT_CONSTEXPR bool float_signbit(T value) {
```

Comment:
nit: Let's call it just `signbit` without mangling and refer to it as `detail::signbit` to make it clear that it is just a wrapper around `std::signbit`.

(and the same for `is_infinite_fast_float`.

Reply:
I hope that I got you right here since I change `is_infinite_fast_float` and `is_finite_fast_float` in the same manner, by making them just wrappers, I mean.

Reply:
LGTM